src/components/CallView/shared/LocalVideo.vue

```diff
@@ -205,23 +205,40 @@ export default {
 		localStreamVideoError: {
 			immediate: true,
 
-			handler: function(localStreamVideoError) {
-				if (localStreamVideoError) {
-					showError(t('spreed', 'Error while accessing camera'), {
-						timeout: TOAST_PERMANENT_TIMEOUT,
-					})
+			handler: function(error) {
+				if (error) {
+					if (error.name === 'NotAllowedError') {
+						this.notificationHandle = showError(t('spreed', 'Access to camera was denied'))
+					} else if (error.name === 'NotReadableError' || error.name === 'AbortError') {
+						// when camera in use, Chrome gives NotReadableError, Firefox gives AbortError
+						this.notificationHandle = showError(t('spreed', 'Error while accessing camera: it is likely in use by another program'), {
+							timeout: TOAST_PERMANENT_TIMEOUT,
+						})
+					} else {
+						console.error('Error while accessing camera: ', error.message, error.name)
+						this.notificationHandle = showError(t('spreed', 'Error while accessing camera'), {
+							timeout: TOAST_PERMANENT_TIMEOUT,
+						})
+					}
 				}
 			},
 		},
 
 	},
 
+	created() {
+		this.notificationHandle = null
```

Comment:
should be in `data() {` so it's always defined and tracked?

Reply:
I put it here as the Vue docs said it's in private space.
I'm fine moving it to data then

Reply:
Let's keep it consistent for now with the rest of our code.